precompiles/precompiles.go

```diff
@@ -0,0 +1,43 @@
+package precompiles
```

Comment:
I think this file can be excluded from test coverage

Reply:
Actually I just included tests in commit [e8c0a5de38ff9458d731f0599d51126471d43a4b](https://github.com/zeta-chain/node/pull/2703/commits/e8c0a5de38ff9458d731f0599d51126471d43a4b)